evalml/utils/gen_utils.py

```diff
@@ -300,18 +311,26 @@ def _convert_woodwork_types_wrapper(pd_data):
         pd_data (pd.Series, pd.DataFrame, pd.ExtensionArray): Pandas data structure
 
     Returns:
-        New pandas data structure (pd.DataFrame or pd.Series) with original data and dtypes that can be handled by numpy
+        Modified pandas data structure (pd.DataFrame or pd.Series) with original data and dtypes that can be handled by numpy
     """
     nullable_to_numpy_mapping = {pd.Int64Dtype: 'int64',
                                  pd.BooleanDtype: 'bool',
                                  pd.StringDtype: 'object'}
-
+    nullable_to_numpy_mapping_nan = {pd.Int64Dtype: 'float64',
```

Comment:
Hmm why does `Int64Dtype` map to `float64` here? Is it because `float64` supports `np.nan` but `int64` doesn't?

Reply:
Yup, that's right 😅